code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -407,7 +407,7 @@
 /*Suffix: Combo of healing, prob gonna get wack REAL fast*/
 /datum/reagent/medicine/c2/synthflesh
 	name = "Synthflesh"
-	description = "Heals brute and burn damage at the cost of toxicity (66% of damage healed). Touch application only."
+	description = "Heals brute and burn damage at the cost of toxicity (66% of damage healed). 100U or more can restore corpses husked by burns. Touch application only."
```

Comment:
```suggestion
	description = "Heals brute and burn damage at the cost of toxicity (66% of damage healed). 100u or more can restore corpses husked by burns. Touch application only."
```

If possible, I would do this in `/New()` so that if we change the value of 100 the description changes with it.

Reply:
Tried to do this and got help from like 3 maintainers on discord but it doesn't seem this is possible right now without reworking the dispenser.